NineChronicles.Standalone/BlockChainService.cs

```diff
@@ -125,5 +130,18 @@ public UnaryResult<bool> IsTransactionStaged(byte[] txidBytes)
                 isStaged ? "staged" : "not staged");
             return UnaryResult(isStaged);
         }
+
+        public UnaryResult<bool> ReportException(string code, string message)
+        {
+            switch (code)
+            {
+                case "26":
```

Comment:
It should treat the code as shared enum in other pull request.

Reply:
I guess we should handle `27` too because it's also action timeout.